proposals/ard_normalized_radar_backscatter.json

```diff
@@ -61,7 +61,7 @@
         }
     ],
     "returns": {
-        "description": "Backscatter values expressed as gamma0. The data returned is CARD4L compliant and contains metadata.\n\nBy default, the backscatter values are given in linear scale.",
+        "description": "Backscatter values expressed as gamma0 in linear scale.\n\nIn addition to the bands `contributing_area` and `ellipsoid_incidence_angle` that can optionally be added with corresponding parameters, the following bands are always added to the data cube:\n\n- `mask`: A data mask that indicates which values are valid (1), invalid (0) or contain no-data (null).\n- `local_incidence_angle`: A band with DEM-based local incidence angles in degrees.\n\nThe data returned is CARD4L compliant with corresponding metadata.",
```

Comment:
local_incidence_angle -> @clausmichele you mentioned that SNAP/pyroSAR can produce this as a separate output, do I remember this correctly?

Reply:
yes, see here https://pyrosar.readthedocs.io/en/latest/pyroSAR.html?highlight=localIncidenceAngle#pyroSAR.snap.util.geocode